HISTORY.md

```diff
@@ -20,6 +20,7 @@
 * Fix data race bug in hash linked list memtable. With this bug, read request might temporarily miss an old record in the memtable in a race condition to the hash bucket.
 * Fix a bug that `best_efforts_recovery` may fail to open the db with mmap read.
 * Fixed a bug where blobs read during compaction would pollute the cache.
+* Fixed a bug where a data race can happens for setting flags (e.g. `SetIsInSecondaryCache()`) after a handle from secondary cache is `Promote()`.
```

Comment:
HISTORY is intended for API users. This text is about implementation details. What is the user perspective on this bug fix? Who is potentially affected? (only secondary cache users, I believe)

Reply:
Update HISTORY.md accordingly.